packages/sanity/playwright-ct/tests/utils/testHelpers.tsx

```diff
@@ -91,5 +90,14 @@ export function testHelpers({
 
       await new Promise((resolve) => setTimeout(resolve, delay || DEFAULT_TYPE_DELAY))
     },
+    /**
+     * Will create a keyboard event of a given meta/hotkey combination
+     * @param key
+     */
+    useHotKey: async (hotkey: string, metaKey: string | undefined = undefined) => {
```

Comment:
Nice catch! Can we name this something more descriptive to match the others? The `useX` looks like a React hook.

Like `pressHotkey()`? `toggleHotKey()`?

Reply:
Agreed, I have changed it to `toggleHotKey`